webui/react/src/routes/routes.ts

```diff
@@ -1,18 +1,6 @@
 import { RouteConfig } from './types';
 
 const routes: RouteConfig[] = [
-  {
-    id: 'trialLogs',
-    needAuth: true,
-    path: '/experiments/:experimentId/trials/:trialId/logs',
-    title: 'Trial Logs',
-  },
-  {
-    id: 'trialLogs',
-    needAuth: true,
-    path: '/trials/:trialId/logs',
```

Comment:
@hkang1 do you think we can drop the support for this old route?

Otherwise we need to create a component just to handle the redirect for this (need to call API to get the trial to have the experimentId and then redirect).

Reply:
@apizzini should changing line ~25 from:
```
path: '/trials/:trialId',
```
to
```
path: '/trials/:trialId/:tab?',
```
I just tried it and links like `http://localhost:3000/det/trials/2031/logs` worked great.